src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -271,19 +286,41 @@ void Snippet::initSupportedPrimitiveDescriptors() {
         }
 
         impl_desc_type impl_type = impl_desc_type::unknown;
-        if (mayiuse(x64::avx512_core)) {
+#if defined(OPENVINO_ARCH_ARM64)
+        if (dnnl::impl::cpu::aarch64::mayiuse(dnnl::impl::cpu::aarch64::asimd)) {
+            impl_type = impl_desc_type::jit_asimd;
+        }
+#else
+        if (dnnl::impl::cpu::x64::mayiuse(x64::avx512_core)) {
             impl_type = impl_desc_type::jit_avx512;
-        } else if (mayiuse(x64::avx2)) {
+        } else if (dnnl::impl::cpu::x64::mayiuse(x64::avx2)) {
             impl_type = impl_desc_type::jit_avx2;
         }
+#endif
         return {config, impl_type};
     };
 
+    bool isPlanarPrioritized = false;
+#if defined(OPENVINO_ARCH_ARM64)
+    for (size_t i = 0; i < getParentEdges().size(); i++) {
+        const auto& parentNode = getParentEdgeAt(i)->getParent();
+        const auto& desc = parentNode->getSupportedPrimitiveDescriptors();
+        if (parentNode->getType() == Type::FullyConnected || parentNode->getType() == Type::MatMul) {
+            if (desc.size() == 1 && desc[0].getConfig().outConfs[0].getMemDesc()->hasLayoutType(ov::intel_cpu::LayoutType::ncsp)) {
+                isPlanarPrioritized = true;
+            }
+        }
+    }
+#endif
+
+    if (isPlanarPrioritized)
+        supportedPrimitiveDescriptors.emplace_back(initDesc(Planar));
     if (isChannelsFirstApplicable)
         supportedPrimitiveDescriptors.emplace_back(initDesc(ChannelsFirst));
     if (isBlockedApplicable)
```

Comment:
As I know, `ARM` currently doesn't support blocked layout. Could you please set `isBlockedApplicable = false` in `ARM` case?

Reply:
Applied. Thanks Alexandra!